src/PeckaCodingStandard/ruleset.xml

```diff
@@ -45,6 +45,8 @@
 	<rule ref="Generic.PHP.ForbiddenFunctions"/>
 	<rule ref="Generic.PHP.UpperCaseConstant"/>
 
+	<rule ref="../../../../escapestudios/symfony2-coding-standard/Symfony2/Sniffs/Formatting/BlankLineBeforeReturnSniff.php"/>
```

Comment:
Přesunul bych to možná pod `WhiteSpace` případně udělal novou sekci `NewLines`

Reply:
👍